docker/build_artifacts/sagemaker/serve.py

```diff
@@ -155,6 +179,19 @@ def _setup_gunicorn(self):
         log.info("gunicorn command: {}".format(gunicorn_command))
         self._gunicorn_command = gunicorn_command
 
+    def _download_scripts(self, bucket, prefix):
+        log.info("downloading universal scripts ...")
+        client = boto3.client("s3")
```

Comment:
do we still need to put region in for the gov regions?

Reply:
Good point. Let me look that up and also ask DLC team how images are released in gov regions.